standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5771,12 +5789,11 @@ public void performTimeOuts() {
         close(rs, stmt, null);
         int numTxnsAborted = 0;
         for(List<Long> batchToAbort : timedOutTxns) {
-          if (abortTxns(dbConn, batchToAbort, true, false, false) == batchToAbort.size()) {
+          if (abortTxns(dbConn, batchToAbort, true, false, false, TxnErrorMsg.ABORT_TIMEOUT) == batchToAbort.size()) {
             dbConn.commit();
             numTxnsAborted += batchToAbort.size();
             //todo: add TXNS.COMMENT filed and set it to 'aborted by system due to timeout'
             Collections.sort(batchToAbort);//easier to read logs
-            LOG.info("Aborted the following transactions due to timeout: {}", batchToAbort.toString());
```

Comment:
above sorting was only needed for the logging, once you removed it, it became redundant

Reply:
line 5790 prints the same message

Reply:
Added `sort()` in the generic logs section.
Line 5790 - This log prints the number of aborted txns. Included this info (number of aborted txns) in the generic logs as well.